qa/tasks/cephfs/fuse_mount.py

```diff
@@ -22,9 +22,9 @@ def __init__(self, ctx, client_config, test_dir, client_id,
         super(FuseMount, self).__init__(ctx=ctx, test_dir=test_dir,
             client_id=client_id, client_remote=client_remote,
             client_keyring_path=client_keyring_path, hostfs_mntpt=hostfs_mntpt,
-            cephfs_name=cephfs_name, cephfs_mntpt=cephfs_mntpt, brxnet=brxnet)
+            cephfs_name=cephfs_name, cephfs_mntpt=cephfs_mntpt, brxnet=brxnet,
+            client_config=client_config)
 
-        self.client_config = client_config if client_config else {}
```

Comment:
````
Traceback (most recent call last):
  File "../qa/tasks/vstart_runner.py", line 1522, in 
    exec_test()
  File "../qa/tasks/vstart_runner.py", line 1421, in exec_test
    mount = LocalFuseMount(ctx=ctx, test_dir=test_dir,
  File "../qa/tasks/vstart_runner.py", line 714, in __init__
    super(LocalFuseMount, self).__init__(ctx=ctx, client_config=None,
  File "/home/jenkins-build/build/workspace/ceph-api/qa/tasks/cephfs/fuse_mount.py", line 33, in __init__
    self.mount_timeout = int(self.client_config.get('mount_timeout', 30))
AttributeError: 'NoneType' object has no attribute 'get'
````

Need to set the default perhaps in the constructor arguments so we can dodge this.

Reply:
Hmm... right, I'll look into this again and pay attention to the CI checks.  Thanks